src/main/java/edu/hm/hafner/analysis/parser/violations/SarifAdapter.java

```diff
@@ -40,6 +40,12 @@ void extractAdditionalProperties(final IssueBuilder builder, final Violation vio
         }
     }
 
+    @Override
+    boolean isValid(final Violation violation) {
+        return !(violation.getSpecifics().containsKey("suppressed")
+            && violation.getSpecifics().get("suppressed").equals("true"));
```

Comment:
SImplify with `getOrDefault("suppressed", "false")`

Reply:
Fair point! updated to match